src/chapters/functions/exercises.rst

```diff
@@ -0,0 +1,156 @@
+Exercises: Functions
+=====================
+
+Fun Functions
+--------------
+
+1. Define a function with one parameter, which will be a string. The function
+   must do the following:
+
+   a. Have a clear, descriptive name.
+   b. Return only the last character from strings with lengths of 3 or less.
+   c. Return only the first 3 characters from strings with lengths larger than
+      3.
+   d. `Build your first function here <https://repl.it/@launchcode/FunctionsExercises01>`__.
+
+|
+
+1. (Continued) Now test your function:
+
+   e. Outside of the function, define the variable ``str`` and initialize it with
+      a string (e.g. ``'Functions rock!'``).
+   f. Define a second variable and initialize it with
+      ``someNameThatIChose = myFunctionName(str);``.
```

Comment:
I recommend adding `let` to the declaration. Some students may do this on their own, but others will likely copy/paste or type it verbatim.

Reply:
How about just using the prompt ``myFunctionName(str)``?